zerver/openapi/zulip.yaml

```diff
@@ -17079,6 +17164,64 @@ components:
         - to
         - topic
         - content
+    ScheduledMessage:
+      type: object
+      description: |
+        A dictionary for representing a scheduled message.
+      properties:
+        message_id:
+          type: integer
+          description: |
+            The unique ID of the scheduled message. It can be used to modify and
```

Comment:
This ID doesn't live in the same namespace as the IDs of already-sent messages, does it? IIUC the discussion elsewhere, this is an unrelated namespace — it may not correspond to any message now or ever, and it may happen to be the same ID number as some actual message, in which case that message will (except by coincidence) be totally unrelated.

Given that, it'd be good to give the property a name other than `message_id`.

Just `id` would be fine, if you want to rely on the context to say what sort of thing the ID is identifying. Alternatively if you want the name to be explicit, it should be something like `scheduled_message_id` so that it doesn't appear to be explicitly saying something that isn't actually intended.

Reply:
Good catch. Let's change this to `scheduled_message_id`; I think probably having an explicit token like that is most helpful.